webapp/tests/forms/steps/lotse/test_pauschbetrag_steps.py

```diff
@@ -16,31 +20,28 @@ def test_if_person_a_requests_pauschbetrag_is_given_then_validation_should_be_su
                 StepPauschbetragPersonA.name, True, ImmutableMultiDict(form_data))
             form = step.render_info.form
             assert form.validate() is True
-            
-    def test_if_person_a_requests_pauschbetrag_is_not_given_then_validation_should_be_false(self, new_test_request_context):
-        form_data = {}
-        stored_data = {'person_a_has_disability': 'yes', 'person_a_has_pflegegrad': 'yes'}
-        with new_test_request_context(form_data=form_data, stored_data=stored_data, method='POST'):
+                        
+    def test_if_precondition_person_a_has_disability_yes_is_not_satisfied_return_should_be_a_redirect_to_person_a_has_disability(self, new_test_request_context):
+        data = MultiDict({'person_a_has_disability':'no'})
+        with new_test_request_context(form_data=data):
             step = LotseStepChooser().get_correct_step(
-                StepPauschbetragPersonA.name, True, ImmutableMultiDict(form_data))
-            form = step.render_info.form
-            assert form.validate() is False
-            
+                StepPauschbetragPersonA.name, True, ImmutableMultiDict(data))
+            assert step.redirection_step_name == StepDisabilityPersonA.name
```

Comment:
We shouldn't test this here but in the functional test

Reply:
There are a lot more tests like this in this file. Maybe we could have another look at those, too?